doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1596,17 +1941,20 @@ Backtraces
    to find out what went wrong. It is disabled by default for performance
    reasons.
 
-Info trace
-~~~~~~~~~~
+Tracing execution
+~~~~~~~~~~~~~~~~~
 
-.. cmd:: Info @num @ltac_expr
+.. cmd:: {? @attributes } {? @toplevel_selector : } Info @num @ltac_expr {| . | ... }
    :name: Info
 
-   This command can be used to print the trace of the path eventually taken by an
+   Prints the trace of the path eventually taken by an
    |Ltac| script. That is, the list of executed tactics, discarding
    all the branches which have failed. To that end the :cmd:`Info` command can be
    used with the following syntax.
 
+   The command doesn't follow the syntax convention used by other commands.  To be precise, the
+   syntax shown above is as for an additional production for :token:`sentence`.  The command is only
+   valid in proof mode.
```

Comment:
I think this presentation is too heavy and doesn't bring much (especially considering that this command doesn't support any attribute at the current time!).  I would instead leave the `.. cmd::` exactly as it was before, and adapt the paragraph explaining that this command is different to say "This command is only valid in proof mode and can be used wherever a top-level tactic would be accepted. Therefore, it accepts goal selectors." (No need to remind about `...`.)